modules/gov/msgs.go

```diff
@@ -64,21 +63,9 @@ func (msg MsgSubmitProposal) ValidateBasic() sdk.Error {
 	}
 	////////////////////  iris begin  ///////////////////////////
 	if msg.ProposalType == ProposalTypeParameterChange {
-
 		if len(msg.Params) == 0 {
 			return ErrEmptyParam(DefaultCodespace)
 		}
-
-		for _, param := range msg.Params {
-			if p, ok := params.ParamSetMapping[param.Subspace]; ok {
-				if _, err := p.Validate(param.Key, param.Value); err != nil {
-					return err
-				}
-			} else {
-				return ErrInvalidParam(DefaultCodespace, param.Subspace)
-			}
-		}
-
 	}
```

Comment:
Since we cannot validate the param value range in the ValidateBasic(). Maybe we should make the check in the client.

Reply:
I will do it in client and lcd.